src/garage/torch/distributions/diagonal_gaussian.py

```diff
@@ -0,0 +1,78 @@
+import torch
+from math import pi, e
+from garage.torch.distributions.base import Distribution
+
+
+class DiagonalGaussian(Distribution):
```

Comment:
As the previous comment, we just need to use PyTorch distribution.

Reply:
I'll remove the Distribution class and use Pytorch distributions.